packetbeat/docs/fields.asciidoc

```diff
@@ -10021,8 +10059,17 @@ type: keyword
 
 --
 
+*`tls.detailed.server_certificate.issuer.locality`*::
++
+--
+Locality.
+
+type: keyword
+
+--
+
 
-*`tls.server_certificate.fingerprint.md5`*::
+*`tls.detailed.server_certificate.fingerprint.md5`*::
```

Comment:
Should be in `tls.server.hash.md5` and copied to `related.hash`

Reply:
Removed